openff/toolkit/topology/molecule.py

```diff
@@ -1956,11 +1956,23 @@ def are_isomorphic(
             [Dict[int,int]] ordered by mol1 indexing {mol1_index: mol2_index}
             If molecules are not isomorphic given input arguments, will return None instead of dict.
         """
+        import networkx as nx
 
-        def _object_to_n_atoms(obj):
-            import networkx as nx
+        if isinstance(mol1, nx.Graph) and isinstance(mol2, nx.Graph):
+            pass
```

Comment:
Since we allow this method to take in graphs and want to let `FrozenMolecule` subclasses through, I thought it'd be best to explicitly handle this case separately